packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -233,7 +262,7 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
                                         "managedClasses.explorer_paneTitleContainer"
                                     )}
                                 >
-                                    <Label>Component Explorer</Label>
+                                    <Heading size={HeadingSize._6}>Explore</Heading>
```

Comment:
is this the expected title?

Reply:
Thats whats in the Figma files:
![image](https://user-images.githubusercontent.com/37851214/62883148-40605380-bce8-11e9-9f2b-968b61c3f5a6.png)

Reply:
Updated to "FAST Explorer" after talking to @bheston